docs/GCSBucketCreate-action.md

```diff
@@ -36,6 +36,10 @@ It can be found on the Dashboard in the Google Cloud Platform Console.
 
 **Location:** The location where the gcs buckets will get created. This value is ignored if the bucket already exists.
 
+**Encryption Key Name:** It is used to encrypt data written to bucket that will be created by pipeline.
```

Comment:
Used to encrypt data written to any bucket created by the plugin. If the bucket already exists, this is ignored.

Reply:
still have this comment